README.md

```diff
@@ -311,6 +311,139 @@ export default function() {
 }
 ```
 
+#### Locator API
+
+We suggest using the Locator API instead of the low-level
+ElementHandle methods. An element handle can go stale if
+the element's underlying frame is changed. However,
+with the Locator API, even if the underlying frame
+changes, locators will work.
+
+Here's an example to better understand the differences.
+
+```js
+import launcher from "k6/x/browser";
+
+export default function () {
+  const browser = launcher.launch('chromium', {
+    headless: __ENV.XK6_HEADLESS ? true : false,
+  });
+  const context = browser.newContext();
+  const page = context.newPage();
+  page.goto("https://test.k6.io/flip_coin.php", {
+    waitUntil: "networkidle",
+  });
+
+  /*
+  In this example, we will use two locators, matching a
+  different betting button (head and tail) on the page.
+
+  If you were to query the elements once and save them as
+  below, you would see an error after the initial navigation.
+  Try it!
+
+    const head = page.$("input[value='Bet on heads!']");
+    const tail = page.$("input[value='Bet on tails!']");
+
+  The Locator API allows you to get a fresh element handle each
+  time you use one of the locator methods. And, you can carry a
+  locator across frames. Let's create two locators each locates
+  a button on the page.
+  */
+  const head = page.locator("input[value='Bet on heads!']");
+  const tail = page.locator("input[value='Bet on tails!']");
+
+  const currentBet = page.locator("//p[starts-with(text(),'Your bet: ')]");
+
+  // the tail locator clicks on the tail button by using the locator's
+  // initial selector above ("input[value='Bet on heads!']")
+  tail.click();
+  // Since clicking on each button causes page navigation,
+  // waitForNavigation is needed. It's because the page
+  // won't be ready until the navigation completes.
+  page.waitForNavigation();
+  console.log(currentBet.innerText());
+
+  // the head locator clicks on the head button by using the locator's
+  // initial selector above ("input[value='Bet on tails!']")
+  head.click();
+  page.waitForNavigation();
+  console.log(currentBet.innerText());
+
+  tail.click();
+  page.waitForNavigation();
+  console.log(currentBet.innerText());
+
+  page.close();
+  browser.close();
+}
+```
+
+The Locator API can help you abstract a page to simplify testing it. To
+do that, you can use a pattern called the Page Object Model. Here's an
+example.
```

Comment:
I'm not sure if we should include this example in the README. It's a good example and we should have it in the `examples` directory, I just think it's redundant to have two Locators API examples in a README already overloaded with examples.

Reply:
What about linking to it from the README?